install_config/install/disconnected_install.adoc

```diff
@@ -171,7 +171,20 @@ To sync the container images:
 # systemctl start docker
 ----
 
-. Pull all of the required {product-title} containerized components.
+. If you are performing a xref:rpm_vs_containerized.adoc[containerized install],
+pull all of the required {product-title} host component images.
+ifdef::openshift-enterprise[]
+Replace `<tag>` with `{latest-tag}` for the latest version.
+endif::[]
++
+----
+# docker pull registry.access.redhat.com/rhel7/etcd
+# docker pull registry.access.redhat.com/openshift3/ose:<tag>
+# docker pull registry.access.redhat.com/openshift3/node:<tag>
+# docker pull registry.access.redhat.com/openshift3/openvswitch:<tag>
+----
+
+. Pull all of the required {product-title} on-cluster infrastructure component images.
```

Comment:
Maybe drop "on-cluster" to just "infrastructure component images"?

Reply:
Fine with me. I was trying to distinguish between things that run as OpenShift pods on top of a cluster versus those that run directly via docker/runc to bootstrap the cluster itself. I'll leave it to you to come up with something less confusing if you can :)